api/src/identity-access-management/domain/models/RefreshToken.js

```diff
@@ -0,0 +1,23 @@
+import { randomUUID as generateUUID } from 'node:crypto';
+
+import { config } from '../../../shared/config.js';
+
+export class RefreshToken {
+  constructor({ token, userId, scope, source }) {
+    this.token = token;
+    this.userId = userId;
+    this.scope = scope;
+    this.source = source;
+  }
```

Comment:
> Éventuellement chercher à trouver un autre nom pour la classe qui permettrait de différencier la classe et l'attribut `token`. Pas évident …

À la réflexion et avec du recul, je préférerais que cette classe garde bien son nom `RefreshToken`, mais que ce soit l'attribut `token` qui soit renommé en `value`, voire `serializedValue` qui serait encore plus parlant :

```suggestion
  constructor({ userId, scope, source, value }) {
    this.userId = userId;
    this.scope = scope;
    this.source = source;
    this.value = value;
    // or else: this.serializedValue = serializedValue;
  }
```

La justification derrière cette proposition est qu'il faut éviter de nommer de manière identique l'objet et sa version « string-ifiée » à sérialiser.

« Un petit peu » comme pour la classe https://developer.mozilla.org/en-US/docs/Web/API/URL qui n'a pas d'attribut s'appellant `url` et pour laquelle on récupère la valeur sérialisée par une méthode `toString` : `url.toString()`